utilities/provider_tallies/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[project]
+name = "provider_tallies"
+version = "0.0.0"
+description = "Provider tallies scripts for Openverse"
+authors = [
+  {name = "Openverse Contributors", email = "openverse@wordpress.org"},
+]
+dependencies = [
+    "redis ~= 5.0",
```

Comment:
In the API, we've followed the `>=, <` convention for packages. Not suggesting that this is wrong or anything but I find that easier to understand and using that would be more consistent.

Reply:
Thank you for context, that's really helpful! I'll update all the version specifications